packages/react/__tests__/src/components/Popover/index.js

```diff
@@ -267,7 +256,7 @@ test('variant="prompt" should return no axe violations', async () => {
 });
 
 test('should return no axe violations', async () => {
-  const wrapper = mount(<Wrapper />);
+  const wrapper = mount(<Wrapper tooltipProps={{ 'aria-label': 'popover' }} />);
```

Comment:
We get an axe violation since the element that is referenced by the `aria-labelledby` element doesn't exist, so I was just trying to get the test here passing for now. It is curious that axe doesn't continue to raise an issue here, given the spec uses `labelledby` over `label`, but I wonder if that's only if the referenced element exists?

We're in a bit of a weird situation here where the non-prompt variant doesn't require the info text (which is where the `aria-labelledby` is getting tacked on to). So it would be somewhat expected here that the author should either pass in an `aria-label` or point to a reference via `aria-labelledby`. So we might want to consider omitting the latter when the former is present.

Reply:
Oh, I see what you're saying now about the non-prompt variant.

> So we might want to consider omitting the latter when the former is present.
Yeah, that sounds like an issue we could make.